From 23a4e5ddf58f5dc307d7c9a44996248fddb25a13 Mon Sep 17 00:00:00 2001
From: jas-per <muzius@gmail.com>
Date: Thu, 5 Dec 2019 01:03:52 +0100
Subject: [PATCH] remove deprecated dh_desktop call (#108)

---
 stdeb/util.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/stdeb/util.py b/stdeb/util.py
index aa38940..b725d48 100644
--- stdeb/util.py	2015-08-02 18:38:18.000000000 +0000
+++ stdeb/util-dh_desktop.py	2020-04-05 22:08:24.327784913 +0000
@@ -839,9 +839,6 @@
         mime_desktop_files = parse_vals(cfg,module_name,'MIME-Desktop-Files')
         if len(mime_desktop_files):
             need_custom_binary_target = True
-            self.dh_desktop_indep_line = '\tdh_desktop'
-        else:
-            self.dh_desktop_indep_line = ''

         #    E. any mime .desktop files
         self.install_file_lines = []
@@ -1471,5 +1468,4 @@
 binary-indep: build
 %(dh_binary_indep_lines)s
 %(dh_installmime_indep_line)s
-%(dh_desktop_indep_line)s
 """
